content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -44,19 +44,32 @@ for your specific needs. The following built-in configuration profiles are curre
 1. **preview**: the preview profile contains features that are experimental. This is intended to explore new features
                 coming to Istio. Stability, security, and performance are not guaranteed - use at your own risk.
 
+1. **ambient**: the ambient profile is designed to help you get started with [ambient mesh](/docs/ops/ambient).
+
+    {{< warning >}}
+    Ambient is currently in [alpha status](/docs/releases/feature-stages/#feature-phase-definitions).
+
+    Please **do not run ambient in production** and be sure to thoroughly review the [feature phase definitions](/docs/releases/feature-stages/#feature-phase-definitions) before use.
+    In particular, there are known performance, stability, and security issues in the `alpha` release.
+    There are also planned breaking changes, including some that will prevent upgrades.
+    These are all limitations that will be addressed before graduation to `beta`.
+    {{< /warning >}}
+
 {{< tip >}}
 Some additional vendor-specific configuration profiles are also available.
 For more information, refer to the [setup instructions](/docs/setup/platform-setup) for your platform.
 {{< /tip >}}
 
 The components marked as &#x2714; are installed within each profile:
 
-|     | default | demo | minimal | remote | empty | preview |
-| --- | --- | --- | --- | --- | --- | --- |
-| Core components | | | | | | | |
-| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istio-egressgateway` | | &#x2714; | | | | | | |
-| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istio-ingressgateway` | &#x2714; | &#x2714; | | | | &#x2714; |
-| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istiod` | &#x2714; | &#x2714; | &#x2714; | | | &#x2714; |
+|     | default | demo | minimal | remote | empty | preview | ambient |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| Core components | | | | | | | | |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istio-egressgateway` | | &#x2714; | | | | | | | |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istio-ingressgateway` | &#x2714; | &#x2714; | | | | &#x2714; | &#x2714; |
```

Comment:
Not necessarily something to add here, but when using the Gateway API, the istio-gateways aren't created. Should we add some comment here? @frankbu, I know you've done a lot of docs around the Gateway API.

Reply:
I don't think the ingressgateway is installed with the ambient profile. Users that want it need to explicitly enable it. See step 4 here: https://preliminary.istio.io/latest/docs/ops/ambient/getting-started/#download
```suggestion
| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istio-ingressgateway` | &#x2714; | &#x2714; | | | | &#x2714; | |
```

Reply:
What Frank is saying, and I missed it as well, is that even though the istio-gateway shows as installed, it it because of the settings included on the command line (--set components.ingressGateways[0].enabled=true --set components.ingressGateways[0].name=istio-ingressgateway), and not part of the profile itself (at least I assume if they were left off the ingress-gateway would not be there).

Reply:
> (at least I assume if they were left off the command line the ingress-gateway would not be there).

Right, you can confirm by noticing it's disabled in the profile: https://github.com/istio/istio/blob/5e4d2a9ad512d43611e790577c5a9ba11cdb6886/manifests/profiles/ambient.yaml#L24

Reply:
Ah, cool. Thanks for including the link out to the actual `ambient` profile yaml! I was trying to find that. 

I have corrected the table in the latest push.